FrontEnd/src/components/landing-page/companies/Companies.jsx

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+import styles from './Companies.module.css';
+import CompanyCard from '../../CompanyCard/CompanyCard';
+import PropTypes from 'prop-types';
+import useSWR from 'swr';
+
+const MainCompanies = ({ isAuthorized }) => {
+  MainCompanies.propTypes = {
+    isAuthorized: PropTypes.bool,
+  };
```

Comment:
nit: Usually `propTypes` are placed outside and after component at the end of the file

Reply:
Thank you for feedback. Done